protoc-gen-swagger/genswagger/template.go

```diff
@@ -1187,3 +1230,70 @@ func extractSwaggerOptionFromFileDescriptor(file *pbdescriptor.FileDescriptorPro
 	}
 	return opts, nil
 }
+
+func swaggerSchemaFromProtoSchema(s *swagger_options.Schema, reg *descriptor.Registry) swaggerSchemaObject {
```

Comment:
Should https://github.com/johanbrandhorst/grpc-gateway/blob/97aefd3e41117e62982ef489dfb516a228a39ea3/protoc-gen-swagger/genswagger/template.go#L231-L245 and other places extracting the schema object be combined with this?

Reply:
Great idea, I'll see if I can make that work.